### PR TITLE
Add documentation server

### PR DESCRIPTION
**Pull request by czystyl**
I added the HTTP server to host documentation locally, also added small steps what's need to be done to work with documentation locally.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -48,6 +48,18 @@ Next, start Expo and follow on-screen instructions:
...
 ```
 
+## Running docs
+
+Documentation is generated from the Reason interface and comments that are included there.
+
+To start working with documentation:
+
+- Ensure that have the odoc installed [odoc](https://github.com/ocaml/odoc)
+  > if you have the problem during installing the odoc with OCaml dependencies, try: `opam pin add odoc.dev git+https://github.com/ocaml/odoc`
```
Comment (grabbou) [inline CONTRIBUTING.md]: *a problem

Comment (grabbou) [reply CONTRIBUTING.md]: Do we need `>`? :D

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -48,6 +48,18 @@ Next, start Expo and follow on-screen instructions:
...
+- build code with: `yarn clean-build` or if you want to build with watch then use: `yarn start`
+- start docs server using: `yarn start-docs`. Script will run the HTTP server and reload after something will change.
```
Comment (grabbou) [inline CONTRIBUTING.md]: something changes

File: package.json [hunk 3]
```diff
@@ -23,6 +24,7 @@
   "devDependencies": {
     "all-contributors-cli": "^5.4.0",
     "bs-platform": "^4.0.0",
+    "live-server": "^1.2.0",
```
Comment (grabbou) [inline package.json]: Nice! I've always used `http-server`, but this one is nice as it has the live-reload capabilities.

File: CONTRIBUTING.md (hunk 1 above)
Comment (czystyl) [reply CONTRIBUTING.md]: It looks better with `>` (imho) :P

**Pull request merged**